gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinTaskRunner.java

```diff
@@ -196,6 +196,11 @@ public GobblinTaskRunner(String applicationName,
     this.services.addAll(suite.getServices());
 
     this.services.addAll(getServices());
+
+    if (ConfigUtils.getBoolean(this.config, GobblinClusterConfigurationKeys.CONTAINER_HEALTH_METRICS_SERVICE_ENABLED, false)) {
```

Comment:
Create a single static constant for the default value for this config, so you don't have to say false in two places?

Reply:
Fixed.